### PR TITLE
xhtml template for album_row

### PR DESCRIPTION
**Pull request by mitchray**

### CODE REVIEW

**Comment by mitchray** [comment]
Adding unit tests

File: resources/templates/album_row.xhtml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </span>
+    </td>
+    <td class="${CLASS_ARTIST}" tal:content="structure ALBUM/getArtistLink">ARTIST</td>
+    <td class="cel_songs optional" tal:content="structure ALBUM/getSongCount">SONG COUNT</td>
+    <td class="cel_year">
+        <tal:block tal:condition="ALBUM/canShowYear" tal:content="structure ALBUM/getYear">
+            YEAR
+        </tal:block>
+    </td>
+    <td class="${CLASS_COUNTER} optional" tal:condition="IS_SHOW_PLAYED_TIMES" tal:content="structure ALBUM/getPlayedTimes">PLAYED TIMES</td>
```
Comment (lachlan-00) [inline resources/templates/album_row.xhtml:34]: ALBUM/getPlayedTimes doesn't exist yet here

Comment (mitchray) [reply resources/templates/album_row.xhtml:34]: Thanks added now

**Comment by lachlan-00** [comment]
final thing missing is `src/Module/Api/Edit/RefreshUpdatedAction.php`
image

when you edit an object in the little edit window it calls this to reload the template. just have to match whats done for artist and we're good.

**Comment by lachlan-00** [comment]
done

**Pull request merged**